src/framing/ciphertext.rs

```diff
@@ -163,6 +163,11 @@ impl MLSCiphertext {
             content: mls_ciphertext_content.content,
             signature: mls_ciphertext_content.signature,
         };
+
+        if cfg!(feature = "test-vectors") {
```

Comment:
If we're changing the code for test vector generation, why not just remove the code below? As far as I can see it's just for an assertion that shouldn't be there anyway.

Reply:
The code here is bad. But there are other parts of the code and tests that rely on this. I don't want to refactor code in this PR but generate test vectors. Changing the logic here requires more code changes.

Reply:
Yeah, this looks like it's fixing the assertion only. The assertion will be removed in #293 anyway.